tests/unit/modules/test_systemd_service.py

```diff
@@ -243,21 +243,27 @@ def test_available(self):
 
         # systemd < 231
         with patch.dict(systemd.__context__, {"salt.utils.systemd.version": 230}):
```

Comment:
Can you move this one to pytest?

Reply:
This is an adjustment of the tests, not a new one, so I prefer for now leave it here. I will move all the new tests in this PR tho.